js/zb.js

```diff
@@ -2449,15 +2449,14 @@ module.exports = class zb extends Exchange {
 
     async transfer (code, amount, fromAccount, toAccount, params = {}) {
         await this.loadMarkets ();
-        const currency = this.currency (code);
         let side = undefined;
         if (fromAccount === 'spot' || toAccount === 'future') {
             side = 1;
         } else {
             side = 0;
         }
         const request = {
-            'currencyName': currency,
+            'currencyName': code,
```

Comment:
I don't believe this is correct. `code` is the CCXT symbol (the unified one), I think you need to do

```
const currency = this.currency (code);
const request = {
    'currency': currency['id'],
    ...
}
```

Check a method like `fetchBorrowRate` for the exact

Reply:
One more correction, if the param-name is `currencyName`, then it's:

```JavaScript
const currency = this.currency (code);
const request = {
    'currencyName': currency['id'],
    ...
};
```